content/about/index.md

```diff
@@ -1,10 +1,30 @@
 ---
+## Configure header of page
 title: "About"
-description: A blog template for Hugo developed by Formspree and available for free.
+description: "A blog template for Hugo developed by Formspree and available for free."
+author: "Emily Cooper"
+headline: "Hi, my name is Emily Cooper.<br> Nice to meet you."
+show_title_as_headline: false
+intro: "I'm a driven 20-something American from Chicago who moves to Paris for an unexpected job opportunity. I've been tasked with bringing an American point of view to a venerable French marketing firm."
+outro: "<i class='fas fa-mug-hot pr2'></i>If my blog has helped you, you can [buy me a coffee](https://ko-fi.com/?gclid=CjwKCAiA_9r_BRBZEiwAHZ_v1y06x-XbErvkpkAfMWRPgIZ5JShPqrs6-RbFHOL5RvgaEAtyQhOFKBoCvRkQAvD_BwE)!"
 date: 2019-02-19T14:47:22-06:00
 draft: false
-layout: standard
-show_title_as_headline: false
+layout: standard-split
+## Configure featured
+featured_label: "Lately"
+number_featured: 2 # pulling from mainSections in config.toml
+number_categories: 0  # set to zero to exclude
+## Configure profile
+profile_left: false
+role: "Social media strategy @ Savoir"
+avatar: avatar.jpg
+avatar_shape: rounded # circle, square, rounded
+show_social: true
+list_label: "Interests"
+links:
```

Comment:
why is this called "links"?

Reply:
an aspirational label 🤣  I'll be adding a list of links but hadn't gotten to that bit yet!

Reply:
ok, no problem :-D